internal/run/convert.go

```diff
@@ -0,0 +1,190 @@
+		return nil
+	}
+
+	// Check for existence of v2 Release (only if Tiller is running)
+	var v2ReleaseFound bool
+
+	listOptions := utils.ListOptions{
+		ReleaseName: c.release,
+		TillerNamespace: c.tillerNS,
+		TillerLabel: c.label,
+	}
+
+	v2Releases, _ := utils.ListReleasesWithKubeConfig(listOptions, c.kubeConfig, c.kubeContext)
+	if len(v2Releases) > 0 {
+		log.Printf("A v2 Release of %s was found", c.release)
+		v2ReleaseFound = true
+	} else {
+		log.Printf("There is no v2 Release of %s to migrate", c.release)
+		return nil
+	}
+
+	// Check for existence of v3 Release
+	var v3ReleaseFound bool
+
+	kc := common.KubeConfig{
+		Context: c.kubeContext,
+		File:    c.kubeConfig,
+	}
+
+	v3cfg, err := v3.GetActionConfig(c.namespace, kc)
+	if err != nil {
+		return err
+	}
+
+	_, err = v3cfg.Releases.Deployed(c.release)
+	if err != nil {
+		log.Printf("No v3 Release of %s found", c.release)
+	} else {
+		log.Printf("A v3 Release of %s was found", c.release)
+		v3ReleaseFound = true
+	}
+
+	if v2ReleaseFound && v3ReleaseFound {
+		return fmt.Errorf("Release %s, has entries both in v2 and v3 format", c.release)
+	}
+
+	return c.cmd.Run()
+}
+
+// Prepare builds the 2to3 convert command
+// 2to3 convert options:
+//     --delete-v2-releases         v2 release versions are deleted after migration. By default, the v2 release versions are retained
+//     --dry-run                    simulate a command
+// -h, --help                       help for convert
+//     --kube-context string        name of the kubeconfig context to use
+//     --kubeconfig string          path to the kubeconfig file
+// -l, --label string               label to select Tiller resources by (default "OWNER=TILLER")
+// -s, --release-storage string     v2 release storage type/object. It can be 'secrets' or 'configmaps'. This is only used with the 'tiller-out-cluster' flag (default "secrets")
+//     --release-versions-max int   limit the maximum number of versions converted per release. Use 0 for no limit (default 10)
+// -t, --tiller-ns string           namespace of Tiller (default "kube-system")
+//     --tiller-out-cluster         when  Tiller is not running in the cluster e.g. Tillerless
+func (c *Convert) Prepare() error {
+
+	args := c.globalFlags()
+	args = append(args, "2to3", "convert")
+
+	// This is a required flag but is always passed by drone-helm
+	args = append(args, "--kubeconfig", c.kubeConfig)
+
+	if c.deleteV2Releases {
+		args = append(args, "--delete-v2-releases")
+	}
+
+	if c.dryRun {
+		args = append(args, "--dry-run")
+	}
+
+	if c.kubeContext != "" {
+		args = append(args, "--kube-context", c.kubeContext)
+	}
+
+	if c.label != "" {
+		args = append(args, "--label", c.label)
+	}
+
+	if c.releaseVersionsMax != 0 {
+		args = append(args, "--release-versions-max", string(c.releaseVersionsMax))
+	}
+
+	if c.tillerNS != "" {
+		args = append(args, "--tiller-ns", c.tillerNS)
+	}
+
+	args = append(args, c.release)
```

Comment:
Should this handle a release name of "" and fail early with a message since it is required?

Reply:
Could be although the plugin will fail if no release is passed, but it makes sense to check empty here as in other flags